spyder/widgets/editor.py

```diff
@@ -965,11 +965,36 @@ def __modified_readonly_title(self, title, is_modified, is_readonly):
             title = "(%s)" % title
         return title
 
-    def get_tab_text(self, filename, is_modified=None, is_readonly=None):
+    def get_tab_text(self, index, is_modified=None, is_readonly=None):

```

Comment:
I finally understood what's this error is all about: you just need to add a dot to the docstring below this line, so that it reads like this

```python
"""Return tab title."""
```